transmogrifier/sources/ead.py

```diff
@@ -123,3 +158,24 @@ def get_source_record_id(cls, xml: Tag) -> str:
             xml: A BeautifulSoup Tag representing a single EAD XML record.
         """
         return xml.header.identifier.string.split("//")[1]
+
+    @classmethod
+    def parse_mixed_value(cls, item: type, skipped_elements: list) -> Generator:
+        """
+        Parse an item in a mixed value of XML elements and strings according to its type.
+        Recursive given the unpredictable structure of EAD values.
+
+        Args:
+            item: An item in a mixed value that may be a BeautifulSoup NavigableString or
```

Comment:
Naming in recursive functions often spins me around so let me know if you can see better var names here